### PR TITLE
Add unit test function to test DeletePod

### PR DESCRIPTION
**Pull request by NickrenREN**
Add unit test function to test DeletePod() of podManager

### CODE REVIEW

**Comment by resouer** [comment]
@k8s-bot ok to test

File: pkg/kubelet/pod/pod_manager_test.go [hunk 1]
```diff
@@ -108,3 +108,58 @@ func TestGetSetPods(t *testing.T) {
 	}
 
 }
+
+func TestDeletePods(t *testing.T) {
+	mirrorPod := &v1.Pod{
+		ObjectMeta: v1.ObjectMeta{
+			UID:       "987654321",
```
Comment (resouer) [inline pkg/kubelet/pod/pod_manager_test.go]: use uuid util, check other tests to see how.

File: pkg/kubelet/pod/pod_manager_test.go [hunk 2]
```diff
@@ -108,3 +108,58 @@ func TestGetSetPods(t *testing.T) {
...
+			Name:      "bar",
+			Namespace: "default",
```
Comment (resouer) [inline pkg/kubelet/pod/pod_manager_test.go]: check other tests to see what namespace they are using

File: pkg/kubelet/pod/pod_manager_test.go [hunk 3]
```diff
@@ -108,3 +108,58 @@ func TestGetSetPods(t *testing.T) {
 	}
 
 }
+
+func TestDeletePods(t *testing.T) {
+	mirrorPod := &v1.Pod{
+		ObjectMeta: v1.ObjectMeta{
+			UID:       "987654321",
+			Name:      "bar",
```
Comment (resouer) [inline pkg/kubelet/pod/pod_manager_test.go]: use a meaningful name, please check other tests for convention.

File: pkg/kubelet/pod/pod_manager_test.go (hunk 2 above)
Comment (NickrenREN) [inline pkg/kubelet/pod/pod_manager_test.go]: I copy these codes from TestGetSetPods() function in the same file

**Comment by NickrenREN** [comment]
Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[pkg/kubelet/pod/pod_manager_test.go, line 115 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39237#-KZyt6m46KCKge1R8qIu-r1-115:-K_XwK7Loobhh-Uj4Gco:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/6d86d7048fe14b1339bed4caab39b56d4c024aee/pkg/kubelet/pod/pod_manager_test.go#L115)):*
<details><summary><i>Previously, resouer (Harry Zhang) wrote…</i></summary><blockquote>

use uuid util, check other tests to see how.
</blockquote></details>

Done.

---

*[pkg/kubelet/pod/pod_manager_test.go, line 116 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39237#-KZyt6m46KCKge1R8qIu-r1-116:-K_XwKSyTqidthFCv_9W:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/6d86d7048fe14b1339bed4caab39b56d4c024aee/pkg/kubelet/pod/pod_manager_test.go#L116)):*
<details><summary><i>Previously, resouer (Harry Zhang) wrote…</i></summary><blockquote>

use a meaningful name, please check other tests for convention.
</blockquote></details>

Done.

---

*[pkg/kubelet/pod/pod_manager_test.go, line 117 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39237#-KZyt6m46KCKge1R8qIu-r1-117:-K_XwMEDd1NbC77BwOyK:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/6d86d7048fe14b1339bed4caab39b56d4c024aee/pkg/kubelet/pod/pod_manager_test.go#L117)):*
<details><summary><i>Previously, NickrenREN wrote…</i></summary><blockquote>

 I copy these codes from TestGetSetPods() function in the same file
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39237)*

**Comment by NickrenREN** [comment]
@resouer   I have modified . Could you help with the review again? Thanks

**Pull request merged**